Source/Tools/LinuxEmulation/LinuxSyscalls/SignalDelegator.cpp

```diff
@@ -1467,6 +1468,20 @@ namespace FEX::HLE {
 #endif
         }
       }
+      else if (Signal == SIGSEGV &&
+               (reinterpret_cast<void*>(ArchHelpers::Context::GetPc(UContext)) == FaultSafeMemcpy::CopyFromUser_FaultLocation ||
+               reinterpret_cast<void*>(ArchHelpers::Context::GetPc(UContext)) == FaultSafeMemcpy::CopyToUser_FaultLocation)) {
```

Comment:
Wrapping these two checks into a single helper function (like `FaultSafeMemcpy::IsFaultLocation(GetPc(UContext))`) keeps implementation details more contained, reducing noise here, in the header file, and in the x86 placeholder path.